core/src/main/kotlin/org/evomaster/core/output/TestCaseWriter.kt

```diff
@@ -428,6 +531,26 @@ class TestCaseWriter {
          *  TODO: get the type from the REST call
          */
         return ".accept(\"*/*\")"
+    }
+
+    private fun handleExpectations(result: RestCallResult, lines: Lines, active: Boolean){
+
+
+
+        lines.add("expectationHandler()")
+        lines.indented {
+            lines.add(".expect()")
+            lines.add(".that(activeExpectations, true)")
```

Comment:
this is bit weird... you asserting both `true` and `false` on `activeExpectations`? if this is indeed intended, please add a comment to clarify it

Reply:
The idea is to have a short example of conditions that fail, but still not cause the test case to fail (unless the expectations are active). 
I've added a comment to reflect this, but this will change quite a lot.